src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import duke.command.AddTaskCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+/**
+ * Encapsulates a parser for Duke. It makes sense
+ * of the inputs supplied by the user.
+ */
+public class Parser {
+
```

Comment:
You can create enum for these static variables as such if you want to: https://www.programiz.com/java-programming/enum-string